net-vpn/i2pd/i2pd-2.52.0.recipe

```diff
@@ -19,10 +19,10 @@ COPYRIGHT="i2pd developers"
 LICENSE="BSD (3-clause)"
 REVISION="1"
 SOURCE_URI="https://github.com/PurpleI2P/i2pd/archive/refs/tags/$portVersion.tar.gz"
-CHECKSUM_SHA256="d7e4fe2c5c3c00a9115f061b797be3d2fc81bb25beddb20a636ae2b0c912ce31"
+CHECKSUM_SHA256="f5fafa700b61d0791d37bd8eee04912582ea5e3f3b1d80ec339bd8158a30995b"
 
 ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
Why disable 32bit here?

Reply:
Because miniupnpc disabled on 32 bit for now (#10579)

Reply:
Sorry for the late reply, miniupnpc has been disabled for gcc2 on 32bit (!x86_gcc2) not for secondary architcture (x86). :)

Reply:
Build is OK, I see linguist and lrelease in the recipe, not stricktly needed here, I guess they are intended for the Qt port?